.travis.yml

```diff
@@ -51,6 +51,6 @@ install:
 
 script:
     - flake8 --count hnn_core
-    - python -m py.test .  --cov=hnn_core hnn_core/tests/ --cov-report=xml
```

Comment:
does the `py.test` interfere with how MPIBackend is set up to run?

Reply:
I guess I was wondering why you had to add the `-s` flag. Wasn't sure if it was for debugging or something necessary to make the tests pass

Reply:
Ah, just saves me time by printing the output. I will remove it before merging. Actually, since this is still failing, I'm changing title to WIP.